installer/ovatools/vic-ova-ui/main.go

```diff
@@ -0,0 +1,222 @@
+// Copyright 2016-2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"encoding/xml"
+	"fmt"
+	"net"
+	"os"
+	"os/exec"
+	"runtime"
+	"strings"
+	"syscall"
+
+	ui "github.com/gizak/termui"
+
+	"github.com/dustin/go-humanize"
+	"github.com/vmware/vic/pkg/version"
+	"github.com/vmware/vmw-guestinfo/rpcvmx"
+	"github.com/vmware/vmw-guestinfo/vmcheck"
+)
+
+const (
+	VT_ACTIVATE   = 0x5606
+	VT_WAITACTIVE = 0x5607
+)
+
+func main() {
+
+	if runtime.GOOS == "linux" {
```

Comment:
There's other linux specific stuff in here, maybe just skip the check?  Or if we want it to run on more than linux, could move the platform specifics to main_linux.go, main_darwin.go, etc.